main.go

```diff
@@ -121,7 +121,11 @@ func getWatchNamespace() (string, error) {
 }
 
 func IsTopologyInUse(ctx context.Context) (bool, error) {
-	kubeClient := kubeutil.KubeClient
+	clientConfig, err := controllers.GetClientConfig()
+	if err != nil {
+		return false, err
+	}
+	kubeClient := kubeutil.InitKubeClient(clientConfig)
```

Comment:
isn't this a [code dup](https://github.com/IBM/ibm-block-csi-operator/blob/ef790973b2550f59b80f66eec50960052aa36624/controllers/ibmblockcsi_controller.go#L230-L235)?

Reply:
opened ticket CSI-4039